README.md

```diff
@@ -11,7 +11,8 @@ ENCODE ATAC-seq pipeline
 * [SLURM](docs/tutorial_slurm.md)
 * [Sun GridEngine (SGE)](docs/tutorial_sge.md)
 * [Local system with docker](docs/tutorial_local_docker.md)
-* [Local system without docker](docs/tutorial_local_conda.md)
+* [Local system with Conda](docs/tutorial_local_conda.md)
+* [Local system with singularity](docs/tutorial_local_singularity.md)
 
```

Comment:
This is a very trivial thing, but the order that we list the instructions (sort of) directs the attention of the user to one solution maybe more than others. In that we prefer the user to run Singularity over conda, and maybe learn Singularity before Docker (and then not have to learn two things on HPC) we might want to instead do:

 * [Local system with singularity](docs/tutorial_local_singularity.md)
 * [Local system with docker](docs/tutorial_local_docker.md)
 * [Local system with Conda](docs/tutorial_local_conda.md)

The user that is looking for Docker will find it, as will the conda. But importantly, singularity won't be missed, or the user will be prompted to ask "What is Singularity?" The alternative (singularity being last) they don't even notice it's there :P

Reply:
I agree with this. singularity will come first :).